spec/features/incoming_mail_handler_spec.rb

```diff
@@ -0,0 +1,79 @@
+describe "Handles incoming email" do
+  let(:proposal) { FactoryGirl.create(:proposal, :with_parallel_approvers) }
+  let(:approval) { proposal.individual_approvals.first }
+  let(:mail) { CommunicartMailer.actions_for_approver(approval) }
+  let(:mandrill_inbound_noapp) { File.read(RSpec.configuration.fixture_path + '/mandrill_inbound_noapp.json') }
+
+  def mandrill_payload_from_message(mail_msg)
```

Comment:
This is a habit from my C days. The "main" stuff at the bottom, like a pyramid.

Happy to change if private-at-the-bottom is the convention.